pin-cushion.js

```diff
@@ -1,3 +1,5 @@
+import { PinCushionAboutApp } from "./about.js";
+import { libWrapper } from "./shim.js";
```

Comment:
I would maybe consider renaming this to `lib-wrapper-shim.js` but I'm not sure what the best practice is here. I think it's OK.

Reply:
Naming it `shim.js` does not make the file's function as obvious as it probably should be (and would be if there was e.g. a lib/libWrapper/ directory structure, but that'd feel like overkill here).
I renamed it to the mentioned `lib-wrapper-shim.js`.